src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -391,6 +391,42 @@ class InsertStatementSegment(BaseSegment):
     )
 
 
+@redshift_dialect.segment(replace=True)
+class CreateSchemaStatementSegment(BaseSegment):
+    """A `CREATE SCHEMA` statement.
+
+    https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_SCHEMA.html
```

Comment:
Looking at the docs there's a bit missing from the end of the grammar here:
`[ schema_element [ ... ] `

The docs aren't very clear for me as to what exactly they represent but if you're familiar with them then would be good to add them in.

If not then just add a TODO note to the end of the docstring mentioning that we haven't added this.